### DISCUSSION TITLE
Welcome to Vue Mini Discussions!

### DISCUSSION THREAD

**Discussion by yangmingshan** [vue-mini/vue-mini#67] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yangmingshan** [comment]
大家好，我叫杨明山，是 Vue Mini 的作者。我是四川人，目前在新加坡工作，偶尔也会出现在成都。工作和开源之外，我喜欢玩主机游戏 🎮、徒步 🥾、打羽毛球 🏸️，偶尔也跑步 🏃，还想尝试一下网球 🎾

大家也都介绍一下自己吧，例如所在的城市，平时的爱好。说不定可以认识一些新朋友。

**Comment by Songyakang** [comment]
大家好, 我是咸鱼康,  目前在深圳工作,  我喜欢宝可梦(骂完记得买), warframe(有肝你就来)

**Reply by yangmingshan** [reply]
我想玩一下宝可梦，你最推荐哪一作入坑？

**Reply by Songyakang** [reply]
永远都是最新版( 宝可梦传说-阿尔宙斯, 或者明年的az

**Reply by Songyakang** [reply]
不过玩的最久的还是被骂的最多的珍钻复刻

**Comment by w2xi** [comment]
大家好，我叫夏影，湖南人，目前在上海板砖，爱好就是家里蹲，然后喜欢电磁炮，不信你看我头像？

**Reply by yangmingshan** [reply]
科普一下电磁炮，拉大家入坑

**Comment by JexLau** [comment]
大家好我是纪年，在深圳工作，第二曲线是独立开发者。 小程序开发原生党，希望小程序不要审核那么严🤪

**Reply by yangmingshan** [reply]
是时候告别原生拥抱 Vue Mini 了

**Comment by wzs28150** [comment]
大家好，我叫汪中帅，在哈尔滨工作，小程序开发原生党，折腾过一个小程序下拉刷新的组件，也喜欢玩主机游戏和羽毛球~

**Reply by yangmingshan** [reply]
我一直想去哈尔滨感受一下冰雪世界

**Reply by wzs28150** [reply]
额 冰雪大世界只有小时候去过 。。现在除非有外地朋友来，基本不会去，太冷了哈哈

**Reply by yangmingshan** [reply]
不是说主题乐园，就是那个环境。南方孩子没见过。

**Comment by Aysnine** [comment]
大家好，恭喜vue-mini修成1.0正果，我只是个想用软件改变点什么的npc，希望能做点什么，祝好🏃

**Reply by yangmingshan** [reply]
要不用 Vue Mini 写个 Todolist，或者 Hackernews 吧

**Reply by Aysnine** [reply]
As you wish🤘

**Comment by calandnong** [comment]
大家好我是卡农，目前在深圳工作，然后喜欢打王者、旅游、喝茶，怎么说？有兴趣喝一杯么？

**Reply by yangmingshan** [reply]
VueConf 去深圳感觉真不错
